pkg/pipelineascode/pipelineascode.go

```diff
@@ -136,6 +136,12 @@ func (p *PacRun) startPR(ctx context.Context, match matcher.Match) (*tektonv1.Pi
 	// Add labels and annotations to pipelinerun
 	kubeinteraction.AddLabelsAndAnnotations(p.event, match.PipelineRun, match.Repo, p.vcx.GetConfig())
 
+	// Add annotations to PipelineRuns to integrate with Tekton Results
+	err := kubeinteraction.AddResultsAnnotation(p.event, match.PipelineRun)
+	if err != nil {
+		return nil, err
```

Comment:
Should we return if we encounter this error or just log the error and proceed?

Reply:
I think the failure to add these annotations is not critical to the functionality, since my understanding is that these annotations are used for informational purposes only, and so logging the error may be sufficient. wdyt @khrm @chmouel ?